src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,74 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private final String buyOperationKey = "buy";
+    private final String supplyOperationKey = "supply";
+    private final String resultKey = "result";
```

Comment:
it is great that you made these fields final, but they are not constants yet
constants are final, static, and have names in UPPER_CASE